package.json

```diff
@@ -18,6 +18,7 @@
   },
   "dependencies": {
     "@kazupon/vue-i18n-loader": "0.3.0",
+    "autoprefixer": "^9.4.2",
```

Comment:
Since this is managed by `postcss.config.js` in the SlipStream repo - we should include it there I think.